setup.py

```diff
@@ -21,9 +21,9 @@
 # Note: this is order dependent
 os.environ["PIP_FIND_LINKS"] = "https://download.pytorch.org/whl/cu113/torch_stable.html"
 
-import versioneer
-from setuptools import find_packages
-from setuptools import setup
+import versioneer  # noqa: E402
+from setuptools import find_packages  # noqa: E402
+from setuptools import setup  # noqa: E402
```

Comment:
Why are these needed?

Reply:
This makes it possible for setuptools to find the right versions of pytorch and install them as dependencies, because it needs an alternative PIP repository (download.pytorch.org).

If you try to install Morpheus locally, via pip install -e ., it won't work unless you manually specify PIP_FIND_LINKS on the command line, or we inject it here before setuptools is imported. The #noqa flags are to stop flake from complaining that we have code between imports.

Error: 
```bash
ERROR: Could not find a version that satisfies the requirement torch==1.12.0+cu113 (from morpheus) (from versions: 1.4.0, 1.5.0, 1.5.1, 1.6.0, 1.7.0, 1.7.1, 1.8.0, 1.8.1, 1.9.0, 1.9.1, 1.10.0, 1.10.1, 1.10.2, 1.11.0, 1.12.0, 1.12.1, 1.13.0, 1.13.1)
ERROR: No matching distribution found for torch==1.12.0+cu113
```